openmdao_book/features/experimental/plugins.md

```diff
@@ -0,0 +1,58 @@
+# Working with Plugins
+
+The OpenMDAO plugin infrastructure provides a way for you to discover and use code that can extend the functionality of OpenMDAO.
+
+## Discovery of Plugins
+
+Before you can use a plugin, you have to know that it exists,  There are a couple of places on the web where you might find OpenMDAO plugins.  The first is [github](https://github.com). Github repositories can have *topics* associated with them.  These are basically like keywords.  You can type a topic into the github search field at the top of the main page.  If you choose `All Github` as your search scope, you should then see a page with a table showing different searching sub-scopes.  Look for `Topics` in that table and select it.  The topic will be displayed on the right along with the number of repositories found that match that topic.  If you then click on that topic link you'll be taken to a page containing a list of links to repositories that have that topic.  The table below gives a list of openmdao-related topics and what type of plugin a repository labelled with that topic should contain:
+
+
```

Comment:
This table should be wrapped with a eval-rst block if you don't want to convert it to a markdown list which is fine.
````
```{eval-rst}
your table
```
````

Reply:
fixed